app/src/main/java/org/fossasia/susi/ai/MainApplication.java

```diff
@@ -72,7 +72,7 @@ protected void log(int priority, String tag, @NonNull String message, Throwable
                 if (throwable == null) {
                     Log.e(TAG, message);
                 } else {
-                    Log.e(TAG, throwable.toString());
+                    Log.e(TAG, message + throwable.toString());
```

Comment:
Please think about how the message needs to be formatted

`message + "\n" + throwable.toString()`